website/static/worker.js

```diff
@@ -1,8 +1,10 @@
-/* eslint-env worker */
-/* eslint no-var: off, strict: off */
 /* globals prettier prettierPlugins parsersLocation */
 
-var imported = Object.create(null);
+"use strict";
+
+self.PRETTIER_DEBUG = true;
```

Comment:
This is functional https://deploy-preview-8923--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAucBnVADAOlAhkgRgG4CcMQAaECABxgEtpVlRcAnNiAdwAV2FmKXABsuuAJ7MqAIza4wAazgwAyjXn0oAc2Qw2AVzhU4AW2lwAJhcsAZXNv24tcAGIQ2J3DAbbkIXPowEJQgABYwJsIA6qH08KjqYHAqAnH0AG5x4n5g6CGaqHBsMDxyWp7IAGYihVQAVqgAHgBCcorKKrgmcDaacFU1RiANjSqaWsJwAIr6EPADwrUg6myFbH7SuObCITRsmjBR9BYwocgAHAAMVHsQhVFyNH57cGvp-VQAjrPwpbSC-lQAFooHBLJYQmw4N96FDSk4KkhqoshoUTPRdAZUeNJjM5v0kYMqDAtkcTmckAAmYlyejCcYAYQgJkRIFeAFYQvpCgAVLaCZFLdKGACSUGssBUYH2dAAguKVDBxJMFoUAL5qoA